SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKCloudDeploymentConfiguration.h

```diff
@@ -131,7 +131,9 @@ class SSpatialGDKCloudDeploymentConfiguration : public SCompoundWidget
 	bool IsUsingCustomRuntimeVersion() const;
 	FText GetSpatialOSRuntimeVersionToUseText() const;
 
-	FReply OnGenerateConfigFromCurrentMap();
+	ECheckBoxState IsAutoGenerateConfigEnabled() const;
```

Comment:
Make IsAutoGenerateConfigEnabled, or IsUsingAutoGenerateConfig call each other, if they are supposed to get their information from the same source.